purchase_default_terms_conditions/models/res_config_settings.py

```diff
@@ -6,14 +6,16 @@
 
 class ResConfigSettings(models.TransientModel):
 
-    _inherit = 'res.config.settings'
+    _inherit = "res.config.settings"
 
     default_notes = fields.Text(
-        related='company_id.purchase_note',
+        related="company_id.purchase_note",
         string="Purchase Terms & Conditions",
         default_model="purchase.order",
-        readonly=False)
+        readonly=False,
+    )
 
     use_purchase_note = fields.Boolean(
-        string='Use Purchase Default Terms & Conditions',
-        config_parameter='purchase.use_purchase_note')
+        string="Use Purchase Default Terms & Conditions",
+        config_parameter="purchase.use_purchase_note",
+    )
```

Comment:
This diff is unexpected here - it should have been part of the migration commits.